docs/draft/rest_api.md

```diff
@@ -0,0 +1,34 @@
+# First Draft for the REST API
+> This is only the first draft. Some adaptations are still possible.
+## Data that needs to be provided by the API [GET]
+- current status of the transformer
+  ```
+  GET status/
+  ```
+- the result of the transformation: file/archive
+  ```
+  GET transformation/{id}
+  ```
+- status of the current transformation
+  ```
+  GET transformation/status/{id}
```

Comment:
I think it would be better to put the id parameter in the middle:
`transformation/{id}/status`